packages/api-explorer/src/components/ErrorBoundary/ErrorBoundary.spec.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import { renderWithTheme } from '@looker/components-test-utils'
+import React from 'react'
+import { ErrorBoundary } from './ErrorBoundary'
+
+const ErrorComp = ({ text }: { text: string }) => {
+  const terminateWithPrejudice = () => {
+    const badObj: any = {}
+    return badObj.a.b
+  }
+  return <div>{text + terminateWithPrejudice()}</div>
+}
+
+describe('ErrorBoundary', () => {
+  const header = 'Uh oh! Something went wrong!'
+  const actionMessage = 'Try refreshing the page the correct the problem.'
+  const childText = 'Sweet Child'
```

Comment:
of mine

Reply:
That was the song that was going through my head when I wrote the test :D